src/org/usfirst/frc/team467/robot/Logging.java

```diff
@@ -56,8 +59,17 @@ private static void setupDefaultLogging() {
 			System.out.println("Failed to create log file appender: " + e.getMessage());
 		}
 
+		try {
+			DriverStation station = DriverStation.getInstance();
+			String event = station.getEventName();
+			String type = station.getMatchType().name();
+			String match = Integer.toString(station.getMatchNumber());
+			Logger.getRootLogger().info(String.join(" ", event, type, match));
```

Comment:
I think that the delimiter should probably be something other than just a space to prevent confusion when looking through quickly, but it probably won't matter too much in this case.